python/cudf_polars/cudf_polars/containers/column.py

```diff
@@ -76,12 +78,47 @@ def sorted_like(self, like: Column, /) -> Self:
 
         See Also
         --------
-        set_sorted
+        set_sorted, copy_metadata
         """
         return self.set_sorted(
             is_sorted=like.is_sorted, order=like.order, null_order=like.null_order
         )
 
+    def copy_metadata(self, from_: pl.Series, /) -> Self:
+        """
+        Copy metadata from a host series onto self.
+
+        Parameters
+        ----------
+        from_
+            Polars series to copy metadata from
+
+        Returns
+        -------
+        Self with metadata set.
+
+        See Also
+        --------
+        set_sorted, sorted_like
+        """
+        ascending = from_.flags["SORTED_ASC"]
+        descending = from_.flags["SORTED_DESC"]
+        if ascending or descending:
+            has_null_first = from_.item(0) is None
+            has_null_last = from_.item(-1) is None
```

Comment:
Just for my context, does it matter what the resulting order is if a series has no nulls at all?

Reply:
Is there are no nulls whether nulls were sorted first or last doesn't change the sort order.

One would use this to compute (say) `max` by selecting the last non-null element of an ascending sorted column. To pick the right one if there _are_ nulls you need to know if nulls are at the beginning or the end. If there aren't any nulls, you don't care where they are.